public/src/main/java/com/rackspace/salus/telemetry/api/config/ServicesProperties.java

```diff
@@ -38,4 +37,6 @@
   String agentCatalogManagementUrl = "http://agent-catalog-management:8080";
   @NotBlank
   String authServiceUrl = "http://authserv:8082";
+  @NotBlank
+  String ambassadorServiceUrl = "http://ambassador:8081";
```

Comment:
The`authServiceUrl` one threw off the pattern. With this it'll work as-is in deployments
```suggestion
  String ambassadorServiceUrl = "http://ambassador:8080";
```